src/VisualStudio/TestUtilities/Proxy/WorkspaceProxy.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Reflection;
+using Roslyn.VisualStudio.Test.Utilities.Remoting;
+
+namespace Roslyn.VisualStudio.Test.Utilities.Proxy
+{
+    internal class WorkspaceProxy
+    {
```

Comment:
As above, I feel these should only be called in one location and should be inlined there.